src/layouts/partial-panel-resolver.html

```diff
@@ -74,6 +75,8 @@
         type: Object,
         computed: 'computeCurrentPanel(hass, routeData)',
         observer: 'panelChanged',
+        notify: true,
```

Comment:
I don't think that it makes sense to compute this here and then propagate it all up? It makes more sense to compute this in `<home-assistant>` and pass it down as a property on `hass`